### ISSUE TITLE
exists tactic

### ISSUE THREAD

**Issue by vladimirias** [UniMath/UniMath#583]
While this is a small thing, I would like to raise the following issue for discussion. 

The concept of "exist" in the UF corresponds to the propositional truncation of a type - the interpretation of the existential quantifier in the "logical" version of the Curry-Howard correspondence. 

The use of "exist" as the name of a Coq tactic is based on the "data" version of the Curry-Howard correspondence that is not used in the UF.

Can we rename "exist" into e.g. "given"?

**Comment by DanGrayson** [comment]
It would be easy to define a new tactic with `Ltac` to do that.  But "given" is the English synonym for the tactic `intro`, so I would prefer something like "provide".

**Comment by DanGrayson** [comment]
Here's what I mean.  Suppose my goal is to prove a statement about every finite group.  Then I would introduce G by saying "Given G, ...".

Oh, and in your papers, you use it as a synonym for `forall`:
"Problem 3.1   Given a C-system CC and a presheaf F : CCop → Sets to construct a C0-system that will be denoted CC[F] and called the F-extension of CC."

**Comment by DanGrayson** [comment]
A disadvantage of making that change is that it becomes harder for Coq users to read our code.

**Comment by siddharthist** [comment]
I agree that the `exists` tactic is somewhat semantically misleading, but I think readability for new users is more important.
